### ISSUE TITLE
Console says JQuery is needed

### ISSUE THREAD

**Issue by ToonTalk** [afsheenam/UUI#1]
bootstrap.min.js:6 Uncaught Error: Bootstrap's JavaScript requires jQuery
    at bootstrap.min.js:6

**Comment by afsheenam** [comment]
This is weird. I have a reference to jQuery:

<script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.2.1.min.js"></script>

I will look into it

**Comment by ToonTalk** [comment]
Perhaps you need to re-order them so JQuery is loaded first?

**Comment by afsheenam** [comment]
I don't think so. When I look at the page source for the MetroUI webpages, the order is the same. The difference is that they are using jQuery 2.1.3 and we are using the latest version: 3.2.1
In any case, I will try reordering

**Comment by afsheenam** [comment]
OK, long story: 
We are using bootstrap and Metro UI. The original Metro UI webpages do not use bootstrap directly. Therefore, in their pages the order is correct, in our pages jQuery should be loaded before Bootstrap.
I think it should be possible to use just Metro UI and completely drop bootstrap, but for now I keep both. I am not closing this issue so we can talk about it more, but the next commit will fix the issue by moving the jQuery link to the top.
